fixieai/agents/code_shot_test.py

```diff
@@ -56,37 +59,54 @@ def test_simple_agent_func_calls(dummy_agent):
     fast_api = fastapi.FastAPI()
     fast_api.include_router(dummy_agent.api_router())
     client = testclient.TestClient(fast_api, raise_server_exceptions=False)
+    headers = {"Authorization": "Bearer fixie-test-token"}
 
     # Test Func[simple1]
-    response = client.post("/simple1", json={"message": {"text": "Howdy"}})
+    response = client.post(
+        "/simple1", json={"message": {"text": "Howdy"}}, headers=headers
+    )
```

Comment:
Consider add this assertion to make sure the token extraction from header logic is correct
```suggestion
    )
    agent.verify_token.assert_called_once_with("fixie-test-token")
```